### ISSUE TITLE
store some basic game metadata

### ISSUE THREAD

**Issue by benjaminpjones** [govariantsteam/govariants#306]
Stuff like:

- game created date
- game creator

Would be useful for filtering, and generally interesting to display in the UI.

**Comment by merowin** [comment]
I would like to add: result of finished games. This can be derived from the moves, but it would be helpful as filter in db queries. The timeout-handler is one example that would benefit from it.

**Comment by benjaminpjones** [comment]
Sounds good! For the timeout handler, does it just need to know a game is finished?  Or is the result actually important?

I was thinking "date ended" would be good too.

**Comment by merowin** [comment]
Yeah for the timeout handler, the result is irrelevant, only if the game is finished or not.
